tt_metal/common/bfloat16.hpp

```diff
@@ -18,20 +18,13 @@ class bfloat16 {
     uint16_t uint16_data;
 
 public:
-    static const size_t SIZEOF = 2;
```

Comment:
Looks like removing this causes build failures when building tests.

also maybe it impacts kernels...

What about: static constexpr size_t SIZEOF = 2;